network/player_test.go

```diff
@@ -69,3 +65,119 @@ func TestConvertToCreatePlayerResponse(t *testing.T) {
 		assert.Equal(t, tc.expResp, resp, tc.desc)
 	}
 }
+
+func TestConvertToGetActiveGamesForPlayerResponse(t *testing.T) {
```

Comment:
Should we test some games where Alice isn't playing, or does `ConvertToGetActiveGamesForPlayerResponse` assume the games passed in are already filtered by current player?